weio/fast_output_file.py

```diff
@@ -212,6 +281,107 @@ def unit(s):
         units = [unit(c) for c in self.data.columns]
         return units
 
+    def _readAll(self):

```

Comment:
This leads to some code duplication. Maybe we could have the standalone function `load_binary_output`  call subfunctions  `load_binary_header` and `load_binary_content`. The class could then call each of these standalone sub functions. Because these are standalone, it would require storing some intermediate variables in a dictionary most likely.